### ISSUE TITLE
It's possible to send RelatedItem objects missing the required Title field

### ISSUE THREAD

**Issue by digitaldogsbody** [datacite/datacite#1709]
### Describe the bug

Related Items must have at least 1 title (per schema 4.4 PDF), however due to an error in the XSD, they have a `minOccurs` of 0, and so will validate when sent as XML, or JSON (which is converted to XML and then validated by the current workflow) even when missing this element.

#### Expected Behaviour

DOIs submitted with a Related Item that does not have at least 1 title should fail to validate.

#### Current Behaviour

DOIs submitted via REST API (JSON or XML data) or XML file upload validate successfully when this data is invalid.

#### Possible Implementation

Ideally we would fix the Schema XSD, but this probably can't/won't happen until 4.5. Until then, we could add a manual check of this field in Bolognese and throw an error when appropriate.

**Comment by KellyStathis** [comment]
@digitaldogsbody Do you think we could fix this in 4.5, or would we have to wait until 5.0? 

We have a similar issue where the XSD doesn't match the docs here: https://github.com/datacite/datacite/issues/1653. However, this issue has unfortunately been around for longer, so it might be trickier to justify a fix in a minor version.

**Comment by digitaldogsbody** [comment]
Yes I think we can fix this in 4.5 - it's a bug fix, even if it's technically a breaking change
